src/vanna/base/base.py

```diff
@@ -734,7 +734,7 @@ def generate_plotly_code(
 
         plotly_code = self.submit_prompt(message_log, kwargs=kwargs)
 
-        return self._sanitize_plotly_code(self._extract_python_code(plotly_code))
+        return self._sanitize_plotly_code(self._extract_python_code(plotly_code.strip()))
```

Comment:
Would make sense to move this `.strip()` as part of the `_sanitize_plotly_code()` or `_extract_python_code()` instead?

Reply:
Of course, I moved it into _extract_python_code(), since I think it's better to address redundant whitespace issues right after the LLM response